plots/scripts/7-plot_plier_pathways.R

```diff
@@ -45,15 +45,27 @@ output_filename <- file.path(output_directory,
                              str_c(file_identifier,
                                    "_PLIER_jaccard.pdf"))
 
+# sample size levels
+sample_size_levels = c("Single Platform\n(half sample size)",
+                       "Combined Array and RNA-seq (full sample size)",
+                       "Single Platform\n(full sample size)")
+
 # Read in data
 jaccard_df <- read_tsv(plot_data_filename,
                        col_types = "dddddcdcddl") %>%
-  filter(pseq == 50) %>%
-  mutate(sample_size = case_when(nmeth == "array_only" ~ "Single Platform",
-                                 nmeth == "seq_only" ~ "Single Platform",
-                                 TRUE ~ "Combine Array and RNA-seq (double sample size)"),
+  #filter(pseq == 50) %>%
```

Comment:
Checking my understanding - we're only at the following %seq levels 0, 50, 100, and 50 is saved as `array_only` and `seq_only` because of what we did in `7-extract_plier_pathways.R` correct?

Reply:
👍 yes -- and now we include full size 0% and 100% RNA-seq (aka full size array and full size RNA-seq)